### PR TITLE
docs: correct the example command to install angular

### PR DESCRIPTION
**Pull request by mangalrajg**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by AndrewKushnir — APPROVED** [review]
@mangalrajg thanks for improving Angular docs!

File: aio/content/guide/setup-local.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ To install the Angular CLI, open a terminal window and run the following command
 
 <code-example format="shell" language="shell">
 
-npm install -g &commat;angular/cli&lt;aio-angular-dist-tag class="pln"&gt;&lt;/aio-angular-dist-tag&gt;
+npm install -g &commat;angular/cli
```
Comment (gkalpak) [inline aio/content/guide/setup-local.md:46]: This should actually be:

```suggestion
npm install -g &commat;angular/cli<aio-angular-dist-tag></aio-angular-dist-tag>
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
Good catch, @mangalrajg :+1:

**Comment by AndrewKushnir** [comment]
Note to the Caretaker: the feedback has been addressed, this PR is ready for merge.

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit 4ad7c2e4df35093a3783ab0b47eabcd0234c850e.

**Pull request closed without merging**